addons/web/static/src/views/calendar/calendar_controller.js

```diff
@@ -116,6 +116,10 @@ export class CalendarController extends Component {
         return this.props.className;
     }
 
+    get editRecordDefaultDisplayText() {
+        return _t("New Event");
```

Comment:
We could add a js test in web to test that method when we override it in custom view.

We could do something like that:
https://github.com/odoo/odoo/pull/120094/files#diff-89c6ed60f280620223e211f7c87cd1a1824696866a6eab08688d620153d781e7R4245